### PR TITLE
MAKE_ENUM and comments cleanup

### PR DESCRIPTION
**Pull request by jonathanlew**
### Description
Found stuff to clean up

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [ ] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [ ] **Remove all commented out code**
- [ ] **Remove extra print statements**: for example, those just used for testing
- [ ] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [ ] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

**Review by kelvinkoon — DISMISSED** [review]

**Review by MathewMacDougall — DISMISSED** [review]

**Comment by jonathanlew** [comment]
found some more fixes please re-review

**Review by MathewMacDougall — DISMISSED** [review]

File: src/software/primitive/move_primitive.h [hunk 1]
```diff
@@ -1,32 +1,15 @@
...
 #include "software/primitive/primitive.h"
+#include "software/util/make_enum/make_enum.h"
 
-enum AutokickType
-{
-    NONE,
-    AUTOKICK,
-    AUTOCHIP
-};
+MAKE_ENUM(AutochickType, NONE, AUTOKICK, AUTOCHIP);
```
Comment (garethellis0) [inline src/software/primitive/move_primitive.h:8]: I think `AutoChickType` might be a bit better, but I'm cool either way, your call.

**Review by garethellis0 — DISMISSED** [review]
LGTM :+1: :tada:

**Review by MathewMacDougall — DISMISSED** [review]

**Review by EvanMorcom — DISMISSED** [review]
LGTM :fire:

**Review by MathewMacDougall — APPROVED** [review]

**Review by kelvinkoon — APPROVED** [review]

**Review by akhilveeraghanta — APPROVED** [review]

**Pull request merged**